.gitignore

```diff
@@ -36,3 +36,6 @@
 
 # Bazel
 bazel-*
+
+# IDE settings
```

Comment:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore so you don't need to do it for every repo your IDE ever touches.